papas/pfobjects.py

```diff
@@ -268,7 +289,14 @@ def __init__(self, tlv, vertex, charge,
                  subtype='s'):
         self.subtype = subtype
         super(Particle, self).__init__(pdgid, charge, tlv)
-        self.uniqueid = Identifier.make_id(Identifier.PFOBJECTTYPE.PARTICLE, subtype)
+        
+    #allow the value used in the particle unique id to depend on the collider type
```

Comment:
do we have something similar for clusters? they should be ordered by pT in pp colliders. 
But I suspect this is going to involve some changes on the cluster class (IIRC it stores only the energy at the moment) - to be discussed.